src/app/components/Containers/TextContainer.test.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import TextContainer from './TextContainer';
+import { isNull } from '../../helpers/tests/testHelpers';
+
+describe('TextContainer', () => {
+  describe('with no data', () => {
+    isNull('should return null', <TextContainer />);
+  });
+});
```

Comment:
Could we add a passing test for both headline and sub-headline? This would ensure that in the future the `BlockType` logic isn't broken.